### DISCUSSION TITLE
Some questions about plugin developing

### DISCUSSION THREAD

**Discussion by ziventian** [Flow-Launcher/Flow.Launcher#2895] [Q&A]
I want to develop plugin for Flow.Launcher with python, but something not works as expect. Here's the question:
1. I wanna copy the result to clipboard, and it does not work with the following code:
```
{
        "Title": title,
        "SubTitle": sub_title,
        "IcoPath": f"images/{icon_path}",
        "JsonRPCAction": {
            "method": "Flow.Launcher.CopyToClipboard",
            "parameters": [copy_params],
            "dontHideAfterAction": True
        }
    }
```
2. how to debug problems like this, is there any debug guide?

**Comment by Garulf** [comment]
I don't see anything wrong with the code you've used as an example. What exactly are you you trying to add to the clipboard?

**Reply by ziventian** [reply]
> I don't see anything wrong with the code you've used as an example. What exactly are you you trying to add to the clipboard?

here is the example:
```
          {
              "Title": "hello query",
              "SubTitle": f'your query is {query}',
              "IcoPath": f"images/app.png",
              "JsonRPCAction": {
                  "method": "Flow.Launcher.CopyToClipboard",
                  "parameters": [f'original query: {query}'],
                  "dontHideAfterAction": True
              }
          }
```
In my expect, if  the query is 'hello', then 'original query: hello'  should be in my clipboard after I click the result or press enter on that result, but nothing happends. My clipboard  does not have that in it.
